apps/nextjs/src/hooks/useTemporaryLessonPlanWithStreamingEdits.ts

```diff
@@ -42,7 +42,7 @@ function patchHasBeenApplied(
 export const useTemporaryLessonPlanWithStreamingEdits = ({
   lessonPlan,
   messages,
-  isStreaming,
+  //isStreaming, // unused now?
```

Comment:
I'm not sure if we need this prop now

Reply:
If commenting doesn't complain, delete!

Reply:
Updated the logic to use the isStreaming prop to reduce additional processing when not streaming

Reply:
I think `isStreaming` is just derived from `isLoading` (from the `ai` hook), but a render behind -- but not worries for now